.github/dependabot.yml

```
@@ -0,0 +1,13 @@
+version: 2
+updates:
+
+  # Check for updates to GitHub Actions every month.
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: monthly
+    # Group updates to a single PR.
+    groups:
+      dependencies:
+        patterns:
+          - '*'
```

This defaults to `auto`, meaning that it would automatically rebase the open PRs if any changer on target branch is detected. Are we sure we want this behavior?
```suggestion
          - '*'
    rebase-strategy: "disabled"      
```

---

Good catch, I think that automatic rebasing is not something that we want, at least for now. Thanks for the heads up! Done in https://github.com/ami-iit/jaxsim/pull/266/commits/144078111dd822ee90f30deb81fc78d4e1224839.